scripts/genappimage.sh

```diff
@@ -0,0 +1,147 @@
+#
+# #URL=$(apt-get install -qq --yes --no-download --reinstall --print-uris libpython2.7 libpython3.2 libperl5.14 liblua5.1-0 libruby1.9.1| cut -d' ' -f1 | tr -d "'")
+# #wget -c $URL
+# #for package in *.deb; do
+# #    dpkg -x $package .
+# #done
+# #rm -f *.deb
+
+
+########################################################################
+# Copy desktop and icon file to AppDir for AppRun to pick them up
+########################################################################
+
+# Download AppRun and make it executable
+#get_apprun
+
+# get_desktop
+
+cp "$ROOT_DIR"/runtime/nvim.desktop "$APP_DIR"
+
+cp "$ROOT_DIR"/runtime/nvim.png "$APP_DIR"
+
+# mkdir -p ./usr/lib/x86_64-linux-gnu
+# copy custom libruby.so 1.9
+# find "$HOME/.rvm/" -name "libruby.so.1.9" -xdev -exec cp {} ./usr/lib/x86_64-linux-gnu/ \; || true
+# add libncurses5
+# find /lib -name "libncurses.so.5" -xdev -exec cp -v -rfL {} ./usr/lib/x86_64-linux-gnu/ \; || true
+
+# copy dependencies
+copy_deps
+
+# Move the libraries to usr/bin
+move_lib
+
+########################################################################
+# Delete stuff that should not go into the AppImage
+########################################################################
+
+# if those libraries are present, there will be a pango problem
+# find . -name "libpango*" -delete
+# find . -name "libfreetype*" -delete
+# find . -name "libX*" -delete
+
+# Delete dangerous libraries; see
+# https://github.com/probonopd/AppImages/blob/master/excludelist
+delete_blacklisted
+
+########################################################################
+# desktopintegration asks the user on first run to install a menu item
+########################################################################
+
+# get_desktopintegration "$LOWERAPP"
+
+########################################################################
+# Determine the version of the app; For use in file filename
+########################################################################
+
+VERSION="Nightly-$COMMIT_DATE-$VIM_VER"
+
+########################################################################
+########################################################################
+# Patch away absolute paths; it would be nice if they were relative
+########################################################################
+
+# patch_strings_in_file "$APP_DIR"/usr/bin/nvim '/usr/local/nvim' '$APPDIR/usr/nvim'
+
+# remove unneeded stuff
+# rmdir ./usr/lib64 || true
+# rm -rf ./usr/bin/*tutor* || true
+# rm -rf ./usr/share/doc || true
+#rm -rf ./usr/bin/vim || true
+# remove unneded links
+# find ./usr/bin -type l \! -name "gvim" -delete || true
+
+########################################################################
+# AppDir complete
+# Now packaging it as an AppImage
+########################################################################
+
+
+cd "$APP_DIR"
+# No need for a fancy script. AppRun can just be a symlink to nvim.
+ln -s ./usr/bin/nvim AppRun
```

Comment:
The cd is necessary in an interactive session: not because ln will start to care, but because tab completion will be enabled and you have less probability of error. This does not apply in scripts.